api/src/main/java/org/openmrs/util/UpgradeUtil.java

```diff
@@ -62,68 +61,81 @@ public static Integer getConceptIdForUnits(String units) {
 				throw new APIException(e);
 			}
 		}
-		
+
 		throw new APIException("upgrade.settings.file.not.have.mapping", new Object[] { units });
 	}
-	
-	public static String getConceptUuid(Connection connection, int conceptId) throws SQLException {
 
-		try (PreparedStatement select = connection.prepareStatement("select uuid from concept where concept_id = ?")) {
+	public static String getConceptUuid(Connection connection, int conceptId) throws SQLException {
+		PreparedStatement select = connection.prepareStatement("select uuid from concept where concept_id = ?");
+		try {
```

Comment:
do not change this here, this has been fixed in
https://issues.openmrs.org/browse/TRUNK-5031

you are currently overriding his fix.